packages/google-maps-mapsplatformdatasets/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@google-cloud/mapsplatformdatasets",
```

Comment:
This should be `@googlemaps/mapsplatformdatasets`, also `mapsplatformdatasets` is a mouthful, perhaps we should add some hyphens in the upstream build file.